hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockDataStreamOutput.java

```diff
@@ -204,7 +205,7 @@ private DataStreamOutput setupStream(Pipeline pipeline) throws IOException {
     //  it or remove it completely if possible
     String id = pipeline.getFirstNode().getUuidString();
     ContainerProtos.ContainerCommandRequestProto.Builder builder =
-        ContainerProtos.ContainerCommandRequestProto.newBuilder()
+        getContainerCommandRequestProtoBuilder()
```

Comment:
The patch is huge (122K), hard to review.  Size could be reduced by prefactoring:
- introduce this factory method
- replace all calls of `newBuilder()`

in a separate patch, without any functional changes, before the fix.

Reply:
ok will raise a patch for this.

Reply:
@adoroszlai @sodonnel I have raised another patch since this patch is becomning too big to review https://github.com/apache/ozone/pull/6812

Reply:
The other PR ,merged now.